lib/utils/log/slog_text_handler.go

```diff
@@ -257,9 +257,6 @@ func formatLevel(value slog.Level, enableColors bool) string {
 	case slog.LevelError:
 		level = "ERROR"
 		color = red
-	case slog.LevelError + 1:
-		level = "FATAL"
-		color = red
```

Comment:
I got rid of support for the fatal level, though I noticed that there are still some residues of it in [teleport.e/tooling/os-package-repo-tool/src/main.go](https://github.com/gravitational/teleport.e/blob/7476cafa19d26dc3046b771c5a585ac9d4fc7d19/tooling/os-package-repo-tool/src/main.go#L101-L104). Should these be removed in favor of just using `slog.LevelError`? I don't quite understand what the comment about preserving compatibility means.